src/cmd/domain.go

```diff
@@ -90,9 +91,14 @@ opslevel list domain
 			common.JsonPrint(json.MarshalIndent(list, "", "    "))
 		} else if isCsvOutput() {
 			w := csv.NewWriter(os.Stdout)
-			w.Write([]string{"NAME", "ID", "ALIASES"})
+			if err := w.Write([]string{"NAME", "ID", "ALIASES"}); err != nil {
+				panic(err)
```

Comment:
its recommended to always `cobra.CheckErr` when inside a cobra command because it properly sets the exit code

Reply:
Most definitely 👍